Source/Scene/ModelExperimental/ModelExperimental.js

```diff
@@ -99,6 +103,40 @@ export default function ModelExperimental(options) {
     defaultValue(options.modelMatrix, Matrix4.IDENTITY)
   );
   this._modelMatrix = Matrix4.clone(this.modelMatrix);
+  this._scale = defaultValue(options.scale, 1.0);
+
+  this._minimumPixelSize = defaultValue(options.minimumPixelSize, 0.0);
+
+  this._maximumScale = options.maximumScale;
+
+  /**
+   * The scale value after being clamped by the maximum scale parameter.
+   * Used to adjust bounding spheres without repeated calculation.
+   *
+   * @type {Number}
+   * @private
+   */
+  this._clampedScale = defined(this._maximumScale)
+    ? Math.min(this._scale, this._maximumScale)
+    : this._scale;
+
+  /**
+   * The true scale of the model after being affected by the model's scale,
+   * minimum pixel size, and maximum scale parameters.
+   *
+   * @type {Number}
+   * @private
+   */
+  this._computedScale = this._clampedScale;
```

Comment:
I see this gets read in the scene graph but I don't think you'd assign to it outside of this class. Should this have a readonly getter? (still marked with `@private`)